Resources/Catalog/PoolXMLFile.py

```diff
@@ -113,15 +116,11 @@ def _getPoolCatalogs( directory = '' ):
       if fname.endswith( '.bak' ):
         gLogger.verbose( 'Ignoring BAK file: %s' % fname )
       elif tarfile.is_tarfile( fname ):
-        try:
-          gLogger.debug( 'Unpacking catalog XML file %s' % ( os.path.join( directory, fname ) ) )
-          tarFile = tarfile.open( os.path.join( directory, fname ), 'r' )
-          for member in tarFile.getmembers():
-            tarFile.extract( member, directory )
+        gLogger.debug( 'Unpacking catalog XML file %s' % ( os.path.join( directory, fname ) ) )
+        with tarfile.open( os.path.join( directory, fname ), 'r' ) as tf:
+          for member in tf.getmembers():
+            tf.extract( member, directory )
             poolCatalogList.append( os.path.join( directory, member.name ) )
-        except Exception, x :
```

Comment:
yes, this is intentional: catching the exception was just printing an error, but then leading to unexpected result. So, in this case is better to just let the exception propagate.